### PR TITLE
Oak

### PR DESCRIPTION
**Pull request by Xeonid**

### CODE REVIEW

File: bugz/DownloadBugzillaUsers.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	}
+
+	// Extract unique IDs from bugs
+	idUserMap, err := getIDS()
+	if err != nil {
+		return
+	}
+
+	// Iterate through the map
+	for id, _ := range idUserMap {
```
Comment (ngortheone) [inline bugz/DownloadBugzillaUsers.go:32]: Rework this loop. 
We no longer need to fetch users from API. Just save each user to json file. I.e.
```
for id, user := range idUserMap {
// save user to file
}
```

**Review by ngortheone — CHANGES_REQUESTED** [review]

File: bugz/getToken.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package usersJson
```
[the remark this answers is not in the document]
Comment (Xeonid) [reply bugz/getToken.go]: ok

File: bugz/DownloadBugzillaUsers.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+func DownloadBugzillaUsers() {
+	apiURL := "https://bugs.freebsd.org/bugzilla/rest/user"
+	token, _ := getToken()
+	// Create a 'users' folder if it doesn't exist
+	err := os.Mkdir("usersJson", os.ModePerm)
```
[the remark this answers is not in the document]
Comment (Xeonid) [reply bugz/DownloadBugzillaUsers.go]: ok

File: bugz/DownloadBugzillaUsers.go (hunk 1 above)
Comment (Xeonid) [reply bugz/DownloadBugzillaUsers.go]: ok

**Pull request closed without merging**